policyengine_us/parameters/gov/states/md/tax/income/credits/senior_tax/two_aged.yaml

```diff
@@ -0,0 +1,24 @@
+description: Senior tax credit given to filers with both filers above the age of 65
+metadata:
+  reference:
+    - title: MD 2022 Resident Booklet
+      href: https://www.marylandtaxes.gov/forms/22_forms/Resident_Booklet.pdf#page=15
+  label: MD Senior Tax Credit
+  unit: currency-USD
+  period: year
+  breakdown:
+    - filing_status
+JOINT:
+  2021-01-01: 0
+  2022-01-01: 1_750
+SINGLE:
+  2021-01-01: 0
+HEAD_OF_HOUSEHOLD:
+  2021-01-01: 0
+  2022-01-01: 1_750
+WIDOW:
```

Comment:
did we confirm that the widow and hoh conditions are the same as joint?

Reply:
yes, for the most part